torch/quantization/observer.py

```diff
@@ -121,15 +142,20 @@ def _calculate_per_channel_qparams(self, min_vals, max_vals):
 
     def _calculate_qparams(self, min_val, max_val):
         # type: (Optional[Tensor], Optional[Tensor]) -> Tuple[Tensor, Tensor]
-        """
-        Given min and max values, this function calculates quantization parameters
+        """Calculates the per tensor quantization parameters, given the min/max.
```

Comment:
why some docstring started with `r` some doesn't?

Reply:
Ideally we want all of them start with `r`. I will make sure they do.

The reason you want them to start with `r` is because when parsed by sphinx, the docstrings that contain `_` or `` ` `` will show some non-sense :)